content/en/blog/_posts/2018-07-09-kubevirt-crds-for-virtualization.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: blog
+title:  'KubeVirt: Extending Kubernetes with CRDs for Virtualized Workloads'
+date:   2018-07-09
+---
+
+**Author**: David Vossel (Red Hat)
+
```

Comment:
If we use this blog post for the 5/6 days of 1.11, we'll want to add a paragraph here referencing the 1.11 features, namely subresources and versions.